index.js

```diff
@@ -26,5 +28,32 @@ module.exports = {
         app.import(app.bowerDirectory + '/prism/components/prism-' + component + '.js');
       });
     }
+
+    // import plugins
+    if (options.plugins){
+      options.plugins.forEach(function(plugin){
+
+        /**
+         * Most Prism plugins contains both a js file and a css file, but there
+         * are exception. `highlight-keywords` for instance, does not have a
+         * css file.
+         *
+         * When the plugin is imported, the app should check for file existence
+         * before calling `app.import()`.
+         */
+
+        // file extensions to be tested for existence.
+        var fileExtensions = ['js', 'css'];
+
+        fileExtensions.forEach(function(fileExtension) {
+          var file = app.bowerDirectory + '/prism/plugins/' + plugin + '/prism-' + plugin + '.' + fileExtension;
+
+          if (fs.existsSync(file)) {
```

Comment:
This is a thing of beauty. Didn't know about the `existsSync` callback. Was looking for something like that, but just went with what I had for the time being.

What is `fs`, where is that from?